internal/cli/atlas/deployments/options/deployment_opts.go

```diff
@@ -247,3 +245,48 @@ func (opts *DeploymentOpts) ValidateAndPromptDeploymentType() error {
 func (opts *DeploymentOpts) IsAtlasDeploymentType() bool {
 	return strings.EqualFold(opts.DeploymentType, AtlasCluster)
 }
+
+func (opts *DeploymentOpts) LocalDeploymentPreRun(ctx context.Context) error {
+	if !localDeploymentSupportedByOs() {
+		_, _ = log.Warningln("Local deployments are not supported on this OS, to see local deployments requirements visit https://www.mongodb.com/docs/atlas/cli/stable/atlas-cli-deploy-local/.")
+	}
+
+	return opts.PodmanClient.Ready(ctx)
+}
+
+func localDeploymentSupportedByOs() bool {
+	os := runtime.GOOS
+	switch os {
+	case "darwin":
+		// MacOS Intel and M1 are supported
+		return true
+	case "windows":
```

Comment:
this should avoid that we use `sysinfo` for windows machines, but there is a test failure I'm investigating

Reply:
updated with a more known library